src/OwaspHeaders.Core.csproj

```
@@ -3,21 +3,28 @@
     <Description>An ASP.NET Core Middleware which adds the OWASP recommended HTTP headers for enhanced security.</Description>
     <VersionPrefix>8.1.0</VersionPrefix>
```

Because NuGet complains if you try to publish a NuGet with the same version (at least, it does for me when I run my GitHub Action to release it). Would you mind upping the patch version for the `VersionPrefix` element?

```suggestion
    <VersionPrefix>8.1.1</VersionPrefix>
```